npm_and_yarn/helpers/build

```diff
@@ -10,5 +10,6 @@ fi
 
 helpers_dir="$(dirname "${BASH_SOURCE[0]}")"
 cd "$helpers_dir"
-cd npm && yarn install && cd -
-cd yarn && yarn install && cd -
+yarn install
+yarn lint
+# yarn test
```

Comment:
We run lint and test in circle so could remove both from here and just do an install.